File: benchmarks/py/fizzbuzz.py
import sys


def fizzbuzz(n: int) -> int:
  acc = 0
  for i in range(1, n + 1):
    a = i % 3 == 0
    b = i % 5 == 0
    if a and b:
      acc += 3
    elif a:
      acc += 1
    elif b:
      acc += 2
  return acc


if __name__ == "__main__":
  n = int(sys.argv[1]) if len(sys.argv) > 1 else 0
  print(fizzbuzz(n))

File: benchmarks/js/fizzbuzz.js
function fizzbuzz(n) {
  // checksum: 0=number, 1=Fizz, 2=Buzz, 3=FizzBuzz
  let acc = 0;
  for (let i = 1; i <= n; i++) {
    const a = i % 3 === 0;
    const b = i % 5 === 0;
    if (a && b) {
      acc += 3;
    } else if (a) {
      acc += 1;
    } else if (b) {
      acc += 2;
    }
  }
  return acc;
}

const n = process.argv[2] ? parseInt(process.argv[2]) : 0;
console.log(fizzbuzz(n));
